server/@types/viewComponents/index.d.ts

```diff
@@ -2,13 +2,8 @@
  * Module declaring types that are used by the nunjucks view components
  */
 declare module 'viewComponents' {
-  import type { Goal } from 'viewModels'
-
-  export interface ActionCardParams {
```

Comment:
`ActionCardParams` was an early version of a type used in an early version of an action card component. It has not been used for some time, so tidying it up 👍